adjust_soiltq/module_bkio_fv3lam_parall.f90

```diff
@@ -187,8 +187,8 @@ subroutine update_soil(this)
     integer :: id,fv3_io_layout_y
     integer :: nlon,nlat,nlat_local,nz
     integer :: i,j,k
-    real(r_kind),allocatable :: r2d8b(:,:)
-    real(r_kind),allocatable :: r3d8b(:,:,:)
+    real,allocatable :: r2d4b(:,:)
+    real,allocatable :: r3d4b(:,:,:)
```

Comment:
line 190-191: why we don't need to use "real(r_single)" here?

Reply:
No reason. Just some code habit.